app/controllers/logs_controller.rb

```diff
@@ -22,6 +29,14 @@ def valid_search_params
     params.keys & %w[ip username location]
   end
 
+  def get_auth_super_admin_requests
+    UseCases::Administrator::GetAuthRequests.new(
+      authentication_logs_gateway: Gateways::Sessions.new(
+        ips: nil
+      )
+    )
+  end
```

Comment:
These duplicated `get_auth_super_admin_requests` and `get_auth_requests` could be simplified to one like:
```
def get_auth_requests
    UseCases::Administrator::GetAuthRequests.new(
      authentication_logs_gateway: Gateways::Sessions.new(
        ips: current_user.super_admin? ? nil : current_organisation.ips.map(&:address)
      )
    )
  end
  ```
 Then you can remove the `if current_user.super_admin?` statement from lines 13 to 23 above.